src/hyperion/parameters/rotation.py

```diff
@@ -21,32 +26,30 @@
     OptionalGonioAngleStarts,
     OptionalXyzStarts,
     RotationAxis,
+    SplitScan,
     TemporaryIspybExtras,
     WithScan,
 )
 from hyperion.parameters.constants import CONST, I03Constants
 
 
-class RotationScan(
-    DiffractionExperimentWithSample,
-    WithScan,
-    OptionalGonioAngleStarts,
-    OptionalXyzStarts,
-):
+class RotationScanCore(OptionalGonioAngleStarts, OptionalXyzStarts):
```

Comment:
Maybe we should rename this, we have `RotationScanCore` and `RotationScanGeneric` which sound very similar.